app/models/event_streamer.rb

```diff
@@ -43,6 +43,7 @@ def start(output)
     @scanner = TerminalOutputScanner.new(output)
     @scanner.each {|event, data| emit_event(event, @handler.call(event, data)) }
   rescue IOError
+  rescue ActionController::Live::ClientDisconnected
```

Comment:
Can just combine the exception catching, thus making it clear '# Raised on stream close' is referring to either exception being thrown rather than 2 empty blocks:
```
rescue IOError, ActionController::Live::ClientDisconnected
  # Raised on stream close
...
```